cmd/actions/install.go

```diff
@@ -24,29 +27,38 @@ var (
 		Long:  "installs a group of services with given namespace and tag version",
 		Args:  cobra.MinimumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
-			_, err := install(args[0], flagNamespace, flagTag, flagDir, flagOutputDir, flagBuild, flagVerbose)
+			_, err := install(args[0], flagNamespace, flagTag, flagDir, flagOutputDir, flagBuild, flagUpgrade, flagVerbose)
 			if err != nil {
 				log.Fatal(err)
 			}
 		},
 	}
 )
 
-func install(group, namespace, tag, workDir, outputDir string, build, verbose bool) (string, error) {
+func install(group, namespace, tag, workDir, outputDir string, buildService, upgrade, verbose bool) (string, error) {
+	cnf := mustNewConfigurd(workDir, tag)
 	sis := cnf.GetServiceItems(namespace, group)
 	if len(sis) == 0 {
 		return "", fmt.Errorf("could not find any service for namespace: %v and group: %v", namespace, group)
 	}
-	if build {
+	if buildService {
 		log.Printf("Building services")
 		for _, si := range sis {
-			_, err := Build(si.Name, tag, workDir, verbose)
+			_, err := build(si.Name, tag, workDir, verbose)
 			if err != nil {
 				return "", err
 			}
 		}
 	}
-	output, err := cnf.Install(log, sis, workDir, outputDir, tag, verbose)
+	output, err := cnf.Install(log, configurd.InstallConfiguration{
+		ServiceItems: sis,
+		BasePath:     workDir,
+		OutputDir:    outputDir,
+		Tag:          tag,
+		Upgrade:      upgrade,
+		Verbose:      verbose,
+	})
```

Comment:
Would it make sense to move InstallConfiguration(rename into something more generic since we would use it for every applicable command) into Configurd without ServiceItems (which dont have anything to do with the configuration, and can be acquired from the instance via namespace and group)? 
This way we can:
 - Encapsulate (install)configuration within Configurd
 - Keep using configurd.New inside PersistentPreRun, and ditch mustNewConfigurd
 - Have access to the (install)configuration fields in the Configurd instance and wont have to pass those around